### DISCUSSION TITLE
Running Migrations the table name are defaulting to all lowercase

### DISCUSSION THREAD

**Discussion by pushyamig** [sequelize/umzug#478] [Q&A] [answered]
Problem: We have named our tables and column names in camel casing like table_name: `accessToken`, `createdAt` etc. When we are running the migrations the UMZUG library is somehow defaulting to table name as `accesstoken` instead of `accessToken`.  We tried the `freezeTableName = true` but still Umzug think the table as `accesstoken` . 
```
Umzug=3.0.0-beta.15
Sequelize=6.3.3
```

The [workaround](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/127/files) we adopted is, name the tables and fields with underscore like `access_token` and `created_at`. What that also mean is we changed the migration table name from `SequelizeMeta` to `app_sequelize_meta`. We did not wanted to do this but we wants things to be consistent naming convention. Here is the [file](https://github.com/ssciolla/canvas-course-manager-next/blob/issue-126-use-underscored/ccm_web/server/src/migrator.ts) with the migration code
But couldn't figure out what is the root cause for it. is anybody encountered this problem?

**Comment by sahilrajput03** [comment, marked as answer]
Is there no answer to this??

**Reply by mmkal** [reply]
Thanks for the bump, I missed this. This isn’t really anything to do with umzug, but whichever sql dialect/implementation you’re using. The OP has the right idea in just using snake case. There are plenty of docs and answers about this, eg Postgres: https://dba.stackexchange.com/a/250949

Umzug is just a script runner that calls the sync method of sequelize at the appropriate time. So it’s _possible_ that there’s a setting in sequelize that can translate table names for you, but it wouldn’t be in the domain of umzug (and personally, I wouldn’t particularly recommend using it over just accepting snake case).
